engine/buff/buff.cpp

```diff
@@ -2068,6 +2148,8 @@ void buff_t::extend_duration( player_t* p, timespan_t extra_seconds )
 
   assert( expiration.size() == 1 );
 
+  extra_seconds = extra_seconds * get_time_modifier_multiplier();
```

Comment:
Didn't have time to check if the time mult is used in all the places it should be, so this comment is just a note for my future self to do so.

Reply:
It's definitely not - it's only used in the bare minimum number of places for it to affect real-time buff durations. Modifying tickrate and showing the apparent duration (as modified by the time rate) are... painful. The knock-on implications of accounting for time modifier rate are kinda profound in downstream code, especially with the difference between apparent duration and real duration. For example, for pandemic refreshing a buff - that relies on the apparent duration, as modified by the time modifier, not the real-time duration. But the real-time duration is what's useful for things like scheduling simulated events and APL logic. And thankfully not currently needed to sim anything, since it's only applied to things that don't tick (anymore).